ci-operator/config/terraform-redhat/terraform-provider-ocm/OWNERS

```diff
@@ -1,9 +1,3 @@
-# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```

Comment:
this one is auto-generated I think
the OWNERS under ``ci-operator/config`` are copied and auto-generated, the OWNERS under ``ci-operator/step-registry`` are not

Reply:
anyway, if jobs are still passing you can leave it as-is. I don't think it matters much

Reply:
It hasn't been generated...

Reply:
no worries. ignore this comment